templates/tab/js/dashboard/src/internal/addNewScopes.js

```diff
@@ -0,0 +1,17 @@
+import { loginAction } from "./login";
+import FxContextInstance from "./singletonContext";
+import { ErrorWithCode } from "@microsoft/teamsfx";
+
+export async function addNewScope(addscopes) {
```

Comment:
please help change the function name to "addNewPermissionScope", along with all the coded refered to this function

Reply:
OK, I've changed the function name and all references.

Reply:
Maybe the file name should be changed too